### PR TITLE
Added downgrade notice to Changelog

### PR DESCRIPTION
**Pull request by nickchase**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note

```

### CODE REVIEW

File: CHANGELOG-1.10.md [hunk 1]
```diff
@@ -199,6 +199,8 @@ In terms of networking, Kubernetes 1.10 is about control. Users now have beta su
 
 ## Before Upgrading
 
+* If you need to downgrade from 1.10 to 1.9.x, downgrade to v1.9.6 to ensure PV and PVC objects can be deleted properly.
```
Comment (msau42) [inline CHANGELOG-1.10.md]: Do we need to care about downgrading from 1.10 to 1.8.x?  With this change, we should recommend they downgrade to 1.9.6 first before 1.8.x.

**Comment by fatsheep9146** [comment]
@nickchase 
May I ask why should we notice this? Does this have related issues? 
> downgrade from 1.10 to 1.9.x, downgrade to v1.9.6

**Pull request merged**